app/build.gradle.kts

```diff
@@ -0,0 +1,220 @@
+plugins {
+    id("com.android.application")
+    id("org.jetbrains.kotlin.android")
+    id("com.google.gms.google-services")
+    id("com.google.devtools.ksp") version "1.7.0-1.0.6"
+    kotlin("kapt")
+    id("com.google.android.libraries.mapsplatform.secrets-gradle-plugin")
+    id("dagger.hilt.android.plugin")
+    id("kotlin-android")
+    id("kotlin-parcelize")
+    id("kover")
+}
+
+val props = rootDir.loadGradleProperties("signing.properties")
```

Comment:
Just out of curiosity, how did you fix the unresolved reference issue here? 😄

Reply:
@AVI5HEK It was my bad. I didn't put the version file in `java` folder. 😄